### DISCUSSION TITLE
Help: entering SPM.mat event files

### DISCUSSION THREAD

**Discussion by sunyunseung1** [HALFpipe/HALFpipe#387] [Q&A] [answered]
Hello,

I am trying to enter SPM.mat event files in halfpipe-1.2.2.
However, when I enter the event files path, an error message 'names' appears.
I guess this message is about the names of conditions in SPM.Sess.U.name.
In my event files, conditions names are entered as shown in the attached image.
I have also attached the err.txt.
I'd appreciate any advice.

![SPM Sess U](https://user-images.githubusercontent.com/99384107/180686347-b6e30729-4095-4dbc-bd1b-68d34b9d39d1.png)
[err.txt](https://github.com/HALFpipe/HALFpipe/files/9179397/err.txt)

Thank you,
Sun

**Comment by HippocampusGirl** [comment, marked as answer]
Hi Sun,

This is currently not supported, but is on our list to implement at some point: https://github.com/HALFpipe/HALFpipe/issues/153

The `.mat`-files that are supported contain three variables, `names`, `onsets`, and `durations`.

Best,
Lea
